src/dialogs/more-info/more-info-controls.js

```diff
@@ -172,6 +184,10 @@ class MoreInfoControls extends LocalizeMixin(EventsMixin(PolymerElement)) {
     return !stateObj || !DOMAINS_NO_INFO.includes(computeStateDomain(stateObj));
   }
 
+  _computeShowRestored(stateObj) {
+    return !stateObj || stateObj.attributes.restored;
```

Comment:
This screen won't show without State obj

Reply:
True, but it is undefined at first render...

Reply:
Right, but then guard it, because now you're implying it's restored entity if state object is not available.
```suggestion
    return stateObj && stateObj.attributes.restored;
```